src/test/scala/fr/polytechnique/cmap/cnam/etl/extractors/ssr/SsrSourceSuite.scala

```diff
@@ -0,0 +1,52 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.ssr
+
+import fr.polytechnique.cmap.cnam.SharedContext
+import fr.polytechnique.cmap.cnam.util.functions.makeTS
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions._
+
+class SsrSourceSuite extends SharedContext with SsrSource {
+
+  lazy val fakeSsrData = {
```

Comment:
Why is this lazy?

Reply:
Because it was not included in the test. I moved it into the test and removed the lazy argument.